command/helper/config.go

```diff
@@ -65,8 +66,9 @@ func DefaultConfig() *Config {
 		DataDir:        "./test-chain",
 		BlockGasTarget: "0x0", // Special value signaling the parent gas limit should be applied
 		Network: &Network{
-			NoDiscover: false,
-			MaxPeers:   50,
+			NoDiscover:       false,
+			MaxOutboundPeers: 10,
+			MaxInboundPeers:  40,
```

Comment:
How did you derive that the default config should be 50 for outbound peers, and 10 for inbound?

Reply:
Milos, it is 40 for inbound and 10 for outbound, 4:1 ratio is the preferred ratio and most of the protocols set 40:10 as default